docs/guide-search.md

```diff
@@ -1,3 +1,127 @@
 # Add search to Gridsome
 
-_Contribute..._
\ No newline at end of file
+## Using [Algolia](https://www.algolia.com/)
+
+Currently Algolia offers upto 50,000 operations for free which should be enough for any site with low traffic. Especially if search isn't used as often.
+
+### Setup
+
+  1. Create a free Algolia account
+  2. Install [gridsome-plugin-algolia](https://gridsome.org/plugins/gridsome-plugin-algolia) eg. `npm install gridsome-plugin-algolia -S`
```

Comment:
depending on how the rest of projects document their plugins, commonly I see this command format as:

```
npm install --save gridsome-plugin-algolia 
```